### PR TITLE
{compiler,lang,tools}[GCCcore/8.4.0] GCC v8.4.0, binutils v2.32, flex v2.6.4, ...

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
taurusi6533.taurus.hrsk.tu-dresden.de - Linux RHEL 7.9, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/d4f3e4092f954d9b2e1f2445caefa911 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/411fb821df6e055c4b7573138eaf7665 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/71b673db5d6437ebe070d311feb2a2ac for a full test report.

**Comment by verdurin** [comment]
Looks okay to me, but I think another maintainer should take a look too.

**Comment by Flamefire** [comment]
Thanks. FTR: I created this to test for compiler bugs in GCC 8.3 which may have been fixed in 8.4. Some were others not. And as this involves more ECs than I thought I created this PR in cases others want to benefit from this too.

File: easybuild/easyconfigs/b/binutils/binutils-2.32-GCCcore-8.4.0.eb
Comment (boegel) [inline easybuild/easyconfigs/b/binutils/binutils-2.32-GCCcore-8.4.0.eb:2]: Any reason for not going with latest binutils  (`2.36`)?

Comment (Flamefire) [reply easybuild/easyconfigs/b/binutils/binutils-2.32-GCCcore-8.4.0.eb:2]: Not really, just wanted to stay close to the others in that version range: 8.3, 9.1, 9.2 all use 2.32. I can replace this by 2.36.1 if you think that's ok

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node2617.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/8b8879cc0ffe8c6dd0e3f6c4ec147d0c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
select-pika-c6gd-2xlarge-0001 - Linux centos linux 8.3.2011, AArch64, ARM UNKNOWN (graviton2), Python 3.6.8
See https://gist.github.com/3ef34008c1f17111b51f60ab890f6d30 for a full test report.

File: easybuild/easyconfigs/b/binutils/binutils-2.32-GCCcore-8.4.0.eb
Comment (boegel) [reply easybuild/easyconfigs/b/binutils/binutils-2.32-GCCcore-8.4.0.eb:2]: I see no reason not to go with the latest binutils for a new (minor) GCC release...

Comment (Flamefire) [reply easybuild/easyconfigs/b/binutils/binutils-2.32-GCCcore-8.4.0.eb]: Done

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node2617.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/0e235190b2ec6d210130daa9c8d7bee3 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (8 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/7a2d6b054d4245a24fc9d303b8ad2e63 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
select-pika-c6gd-2xlarge-0001 - Linux centos linux 8.3.2011, AArch64, ARM UNKNOWN (graviton2), Python 3.6.8
See https://gist.github.com/45d73b4e31075aa8a8a022162c03728d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
easybuild2.novalocal - Linux centos linux 8.3.2011, POWER, IBM pSeries (emulated by qemu) (power9le), Python 3.6.8
See https://gist.github.com/e5f6bcd1f793839bf4a307cd1765599c for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
taurusi8024 - Linux centos linux 7.9.2009, x86_64, AMD EPYC 7352 24-Core Processor (zen2), Python 2.7.5
See https://gist.github.com/6fb1074c829f81b3fb5269f96887a274 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @Flamefire!

**Pull request merged**
